.github/actions/post-slack/action.yml

```diff
@@ -0,0 +1,79 @@
+name: Send message to slack
+
+description: 'Send resuls on slack'
```

Comment:
I had to create that as re-usable workflows cannot be used within steps..

Reply:
why we can't just make it a separate job in the new workflow file `push-important-models.yml`?

Reply:
Because it lead to too many duplicated code, I preferred this approach that made that workflow file much cleaner and easier to understand